### PR TITLE
plan: tiny clean

### PR DESCRIPTION
**Pull request by hanfei1991**

### CODE REVIEW

File: plan/physical_plans.go [hunk 1]
```diff
@@ -157,17 +162,6 @@ func (p *PhysicalMemTable) Copy() PhysicalPlan {
 	return &np
 }
 
-type physicalTableSource struct {
```
Comment (winoros) [inline plan/physical_plans.go]: Why do this?

Comment (hanfei1991) [reply plan/physical_plans.go]: No need to keep a common table source.

File: plan/column_pruning.go [hunk 2]
```diff
@@ -282,15 +282,8 @@ func (p *SelectLock) PruneColumns(parentUsedCols []*expression.Column) {
...
-		used := getUsedList(parentUsedCols, p.schema)
 		for _, cols := range p.children[0].Schema().TblID2Handle {
-			for _, col := range cols {
-				col.ResolveIndices(p.children[0].Schema())
-				if !used[col.Index] {
-					used[col.Index] = true
-					parentUsedCols = append(parentUsedCols, col)
-				}
-			}
+			parentUsedCols = append(parentUsedCols, cols...)
```
Comment (coocood) [inline plan/column_pruning.go]: Can `cols` already in `parentUsedCols`?

**Comment by XuHuaiyu** [comment]
LGTM
/run-all-tests

File: plan/column_pruning.go (hunk 2 above)
Comment (hanfei1991) [reply plan/column_pruning.go]: yes, it doesn't matter.

**Pull request merged**